### ISSUE TITLE
Timeout with latest version

### ISSUE THREAD

**Issue by imperugo** [nathanielvarona/pritunl-client-github-action#74]
### Issue Description
Hi, 
since the last 4 days I'm unable to deploy from my github action due timeout on pritnul client.

I'm getting this

```
Establishing connection: [#####---------------------------------------------] checking 1 out of a maximum of 10 allowed attempts.
Establishing connection: [##########----------------------------------------] checking 2 out of a maximum of 10 allowed attempts.
Establishing connection: [###############-----------------------------------] checking 3 out of a maximum of 10 allowed attempts.
Establishing connection: [####################------------------------------] checking 4 out of a maximum of 10 allowed attempts.
Establishing connection: [#########################-------------------------] checking 5 out of a maximum of 10 allowed attempts.
Establishing connection: [##############################--------------------] checking 6 out of a maximum of 10 allowed attempts.
Establishing connection: [###################################---------------] checking 7 out of a maximum of 10 allowed attempts.
Establishing connection: [########################################----------] checking 8 out of a maximum of 10 allowed attempts.
Establishing connection: [#############################################-----] checking 9 out of a maximum of 10 allowed attempts.
Establishing connection: [##################################################] checking 10 out of a maximum of 10 allowed attempts.
Timeout reached! Exiting...
```

I've solved rolling back to the version 1.1.12 of the client that's fine for now, but I think is important to you to know that something between the 1.1.12 and the lastest one does't work (or better, in my case doesn't).

Thanks

### It has been used with the following parameters:

**Runner Virtual Environments:**
- [ ] Linux
  - [x] Ubuntu 22.04
  - [ ] Ubuntu 20.04
- [ ] macOS
  - [ ] macOS 12
  - [ ] macOS 11
- [ ] Windows
  - [ ] Windows 2022
  - [ ] Windows 2019

**VPN Modes:**
- [x] OpenVPN (ovpn) 
- [ ] WireGuard (wg)

**Client Versions:**
- [ ] Installed from the package manager 
- [ ] Version specific
  
  - [x] 1.3.3637.72

**Start Connection:** *If the connection is started on the setup step.*
- [ ] True 
- [ ] False

**Runner Types:**
- [x] Tested on GitHub-hosted runner 
- [ ] Tested on Self-Hosted runner

  *If it runs on a self-hosted runner:*
  - [ ] Manage Self-hosted
  - [ ] Action Runner Controller (ARC) (a Kubernetes Operator)

**Comment by nathanielvarona** [comment]
Hello @imperugo 

Thank you for using the GitHub Actions I created and raising the first issue from who uses it.
I haven't seen critical issues now with the latest updates I release from the days. My testing works well.

I have two scenarios to ensure that GitHub Actions Metadata syntax works well, such as the `uses:` section.
1. I ran the test on this repository.
https://github.com/nathanielvarona/pritunl-client-github-action/actions/workflows/connection-tests.yml?query=branch:v1
2. I also tested my newly created public test repository that simulated the `uses: nathanielvarona/pritunl-client-github-action@v1`, ensuring it works for anyone.
https://github.com/nathanielvarona/github-actions-uses-public-tests/actions/workflows/pritunl-client-github-action.yaml?query=branch:main

I had experienced this kind of issue before, especially creating a couple number of connections like in my tests.

* When a Pritunl Server `Allow Multiple Devices` is enabled, and multiple established connections you have made in a certain period of time, the Pritunl server might receive `Too Many Authentication Attempts`.

I resolved it by adjusting the Pritunl Server settings
```
sudo pritunl clear-auth-limit
sudo pritunl set app.auth_limiter_ttl 60
sudo pritunl set app.auth_limiter_count_max 100
```

Better try to check first your Pritunl Server logs.
Check out https://docs.pritunl.com/docs/too-many-authentication-attempts

By the way, I updated the issue based on the `Issue Template` I recently merged.
https://github.com/nathanielvarona/pritunl-client-github-action/blob/main/.github/ISSUE_TEMPLATE/issue_template.md

**Comment by imperugo** [comment]
adding @avramenkovladyslav who configured the VPN and knows better than me the information you need.

Thanks

**Comment by imperugo** [comment]
What I've noticed is that with the version 1.1.12 it works after tentative number 11 and the latest versione stop at tentative number 10.

![CleanShot 2023-10-10 at 18 09 58@2x](https://github.com/nathanielvarona/pritunl-client-github-action/assets/758620/6605d1e0-6b5d-447d-b06b-51182bf69ce0)

**Comment by nathanielvarona** [comment]
I see.

On `v1.1.12` and older releases, we have a Connection Timeout of `30` by default.
On your side, the Pritunl Client and VPN Server are usually fully established by `11~` probes.
However, earlier releases I made have a Connection Timeout of `10` by default. 
https://github.com/nathanielvarona/pritunl-client-github-action/compare/v1.1.12...v1.1.23?diff=split#diff-81f73a96810eaaa67034ef131ec342dbf6b4e99f5a2c962b13e08315dc1a822fL15

The resolution is to increase your Connection Timeout.

```
- name: Setup Pritunl Profile
  id: pritunl-connection
  uses: nathanielvarona/pritunl-client-github-action@v1
  with:
    profile-file: ${{ secrets.PRITUNL_PROFILE_FILE }}
    ...
    client-version: 1.3.3637.72
  env:
    CONNECTION_TIMEOUT: 30   # <-- Increase here based on your client-server needs.
```
Reference: https://github.com/nathanielvarona/pritunl-client-github-action#overriding-default-wait-established-connection-timeout

Globalize Environment Variable for `CONNECTION_TIMEOUT` would also work like in my `Connections Tests.`
Example: https://github.com/nathanielvarona/pritunl-client-github-action/blob/main/.github/workflows/connection-tests.yml#L31-L32

**Comment by nathanielvarona** [comment]
Hello again @imperugo,

I recently released the [v1.2.0](https://github.com/nathanielvarona/pritunl-client-github-action/releases/tag/v1.2.0) in which the connection timeout is defaulted to `30.`
If you want to increase it, ignore the use of the Environment Variable and use the new Inputs instead.

Example:
```yaml
- name: Setup Pritunl Profile and Start VPN Connection
  uses: nathanielvarona/pritunl-client-github-action@v1
  with:
    profile-file: ${{ secrets.PRITUNL_PROFILE_FILE }}
    ...
    ...
    established-connection-timeout: '60'
```

Let me know if this works for you Pritunl Infra.

Thanks,

**Comment by avramenkovladyslav** [comment]
@nathanielvarona Thanks, it worked
